.github/workflows/publish-gh-pages.yml

```diff
@@ -0,0 +1,97 @@
+# Sample workflow for building and deploying a Jekyll site to GitHub Pages
+name: Publish site
+
+on:
+  # Runs on pushes targeting the default branch
```

Comment:
nit: gh-pages is not the "default branch", maybe update or remove this comment

Reply:
Fixed